README.md

```diff
@@ -33,11 +33,13 @@ accessible URL for these results. For example:
     {
       "name": "fedora-27",
       "source": "https://download.fedoraproject.org/pub/fedora/linux/releases/27/CloudImages/x86_64/images/Fedora-Cloud-Base-27-1.6.x86_64.qcow2",
+      "upload_results": True,
```

Comment:
Boolean values must be lower-case in JSON: http://www.json.org/

Reply:
Thank you, fixed